From a73259f1797b9ea562c1fede2ebfb6ea5ad9dc3e Mon Sep 17 00:00:00 2001
From: Stefan Eissing <icing@apache.org>
Date: Sat, 2 Dec 2017 18:23:31 +0000
Subject: [PATCH] On the 2.4.x branch:

mod_http2: h2_workers.c now in sync with trunk again and backported as intended.


git-svn-id: https://svn.apache.org/repos/asf/httpd/httpd/branches/2.4.x@1816977 13f79535-47bb-0310-9956-ffa450edef68
---
 modules/http2/h2_workers.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/http2/h2_workers.c b/modules/http2/h2_workers.c
index 2df92b2354..e976df94cf 100644
--- a/modules/http2/h2_workers.c
+++ b/modules/http2/h2_workers.c
@@ -160,7 +160,7 @@ static apr_status_t slot_pull_task(h2_slot *slot, h2_mplx *m)
          * If we still have idle workers, we let the worker be sticky, 
          * e.g. making it poll the task's h2_mplx instance for more work 
          * before asking back here. */
-        slot->sticks = 1;/*slot->workers->max_workers;*/
+        slot->sticks = slot->workers->max_workers;
         return rv;            
     }
     slot->sticks = 0;
-- 
2.17.1

